internal/handlers/image_handler.go

```
@@ -0,0 +1,283 @@
+		"success": true,
+	})
+}
+
+func (h *ImageHandler) PushImage(c *gin.Context) {
+	imageID := c.Param("id")
+
+	var req struct {
+		Tag string `json:"tag"`
+	}
+
+	c.ShouldBindJSON(&req)
```

golangci-lint (1.64.8)

Error return value of `c.ShouldBindJSON` is not checked

(errcheck)